wger/exercises/views/muscles.py

```diff
@@ -57,7 +61,7 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class MuscleAdminListView(LoginRequiredMixin, PermissionRequiredMixin, MuscleListView):
+class MuscleAdminListView(LoginRequiredMixin, PermissionRequiredMixin, ListView):
     '''
```

Comment:
Could you please highlight why you changed inheritance from MuscleListView to ListView yet the implementation done is in MuscleListView.

Reply:
@reifred I made this change because to get a fresh list of muscles in the overview after a delete, we had to switch from the `MuscleListView`  to the `ListView` so as to have the list queried back to the same URL and have our changes effected. without this change, it would be unlikely to get a `reverse match` for our data seeing that the arguments we will not be using an explicit argument.

Reply:
Okay thank you for this clarification

Reply:
@reifred this has been resolved and back to `MuscleListView`